vignettes/qenv.Rmd

```diff
@@ -26,18 +26,17 @@ print(empty_qenv)
 
 ### `qenv` basic usage
 
-The `eval_code()` function executes code within a `qenv` environment, yielding a new `qenv` object as the output. 
+The `eval_code()` function executes code within a `qenv` environment, yielding a new `qenv` object as the output.
 
 ```{r}
-library(magrittr)
```

Comment:
I can see that you have successfully removed this package. In case this is the only usage then we can remove it from the suggest. I'm on the phone so can't check it now lol 😉

Reply:
There is traces of `%>%` in unit tests so I don't think we can't remove this until we rewrite those test cases.